lib/bundles/inspec-compliance/cli.rb

```diff
@@ -169,24 +196,27 @@ def upload(path) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Percei
     desc 'version', 'displays the version of the Chef Compliance server'
     def version
       config = Compliance::Configuration.new
-      info = Compliance::API.version(config['server'], config['insecure'])
-      if !info.nil? && info['version']
-        puts "Chef Compliance version: #{info['version']}"
+      if config['automate']
+        puts 'Version not available when logged in with Automate.'
       else
-        puts 'Could not determine server version.'
-        exit 1
+        info = Compliance::API.version(config['server'], config['insecure'])
+        if !info.nil? && info['version']
+          puts "Chef Compliance version: #{info['version']}"
+        else
+          puts 'Could not determine server version.'
+          exit 1
+        end
       end
     end
 
     desc 'logout', 'user logout from Chef Compliance'
     def logout
       config = Compliance::Configuration.new
-      unless config.supported?(:oidc) || config['token'].nil?
+      unless config.supported?(:oidc) || config['token'].nil? || config['automate']
         config = Compliance::Configuration.new
         url = "#{config['server']}/logout"
         Compliance::API.post(url, config['token'], config['insecure'], !config.supported?(:oidc))
       end
-
```

Comment:
Also we should remove the the server information from the config. This was missing even before